examples/transmissions/handle_exception.py

```diff
@@ -16,12 +16,16 @@
 sp = SparkPost()
 
 try:
-    response = sp.transmissions.send(
-        recipients=['john.doe@example.com'],
-        text='Hello there',
-        from_email='Testing <test@some-domain-you-havent-configured.com>',
-        subject='Testing python-sparkpost exceptions'
-    )
+    response = sp.transmissions.post({
+        'recipients': ['john.doe@example.com'],
+        'content': {
+            'from': 'Testing <test@some-domain-you-havent-configured.com>',
+            'subject': 'Testing python-sparkpost exceptions',
+            'text': 'Hello world!',
+            'html': '<p>Hello world!</p>',
+        },
+        'campaign_id': 'python-sparkpost example',
+    })
 except SparkPostAPIException as err:
     # http response status code
     print(err.status)
```

Comment:
Can we crack open the `SparkPostAPIException` type in this example? It'd be good to show how to recognise HTTP-level errors (e.g. 401 and 5xx) and extract API-level detail to help diagnose.

There's some sample code here: https://github.com/SparkPost/appengine-std-python-sample/blob/master/main.py#L32